packages/twenty-front/src/utils/title-utils.ts

```diff
@@ -19,13 +19,39 @@ export const getPageTitleFromPath = (pathname: string): string => {
     case AppPath.OpportunitiesPage:
       return 'Opportunities';
     case `${AppBasePath.Settings}/${SettingsPath.ProfilePage}`:
-      return 'Profile';
+      return 'Profile - Settings';
```

Comment:
I think we only want to have the values  `Appearance - Settings`, `Profile - Settings` (for all children pages of Profile, except Appearance that is in the product not a child of Profile but that seems to be by the path: settings/profile/appearance), `Accounts - Settings` (for all children pages of Accounts: Emails, Calendars..).
We don't need more granularity that could be hard to maintain as we add more pages/components